Update `Ranking criteria` (#9266)
ppy/osu-wiki: wiki/Ranking_Criteria/en.md

  - **...between 4:15 and 5:00**, the lowest difficulty of each included game mode cannot be harder than an Insane.
  - **[Break times](/wiki/Beatmap/Break) may be combined with [drain time](/wiki/Beatmap/Drain_time) to meet the above thresholds.** For the highest difficulty, this is limited to at most 30 seconds of break time. This does not apply to difficulties with less than 30 seconds of drain time.
  - **osu!mania beatmaps are bound by a different set of drain time thresholds and other spread rules defined in the [osu!mania ranking criteria](/wiki/Ranking_Criteria/osu!mania).**

> - **A beatmap of 2 songs combined together must follow the minimum spread requirements corresponding to the length of the longer song.**[^songs-comp-note] This is to avoid artificial extension of songs as an evasion of time limitations in the beatmap section of these criteria. Exceptions can be made for songs that were exclusively released together.

Reviewer: ```suggestion
- **A beatmap of 2 songs combined together must follow the minimum spread requirements corresponding to the length of the longer song.**[^songs-comp-note] This is to avoid artificial extension of songs as an evasion of the above time limitations. Exceptions can be made for songs that were exclusively released together.
```

- **The audio file of a beatmap must...**
  - **...use the `.mp3` or `.ogg` file format.**
  - **...have an average bit rate no greater than 192 kbps for `.mp3` files, or 208 kbps for `.ogg` files.**
  - **...have an average bit rate no lower than 128 kbps**, if such a source exists. Otherwise, use the highest quality available.
  - **...not be encoded upwards from a lower bitrate.**
- **A beatmap may only contain one song file used by all difficulties.** Multiple song files within a single beatmap are unsupported and result in unexpected behaviour with preview times, metadata, etc.

> - **A beatmap of 2 songs combined together must follow the minimum spread requirements corresponding to the length of the longer song.**[^songs-comp-note] This is to avoid artificial extension of songs as an evasion of time limitations in the beatmap section of these criteria. Exceptions can be made for songs that were exclusively released together.

Reviewer: shouldn't this just be in the beatmap section?

Author: hm, initially went with audio because of the songs comp thing but beatmap might make more sense considering the current wording and how it's more oriented about "spread". feel free to adjust at your own discretion

- **The audio file and hitsound files of a beatmap should not feature any audible and unwarranted sound distortions**, like clipping, muffling, or crackling that is clearly not intended by the artist or part of the song's identity. This is best determined by listening to the audio, rather than using software on its own.
- **The audio file of a song should not be artificially extended in order to meet a time limitation in the beatmap section of this criteria.** This can include (but is not limited to) looping sections of the audio file, lowering the [BPM](/wiki/Music_theory/Tempo) of the song or section of the song, or adding small amounts of music to the song without incorporating it throughout the entire song. This does not apply to [song compilations](/wiki/Beatmap/Song_compilation) or audio files less than the minimum rankable beatmap length.
- **If you do not beatmap the last 20% of your beatmap's audio file, it should be cut.** The intro time is not included. This does not apply if more than 20% of the outro is occupied by a storyboard/video, or if more than 20% of the song's audio is not able to be mapped due to fade-out or timing issues.

> - **A mapset of 2 songs combined together must follow the minimum spread requirements corresponding to the length of the longer song.[^songs-comp-note]** This is to avoid the artificial extension of songs as an evasion of time limitations in the beatmapset section of this criteria. Exceptions can be made for songs that were exclusively released together.

Reviewer: terminology

```suggestion
- **A beatmap of 2 songs combined together must follow the minimum spread requirements corresponding to the length of the longer song.**[^songs-comp-note] This is to avoid artificial extension of songs as an evasion of time limitations in the beatmap section of these criteria. Exceptions can be made for songs that were exclusively released together.
```

[^character-voice-actor]: `CV` (Character Voice) is used when song is sung in character by the character's voice actor. `VO` (Voice Over) is used when a song is sung in character by someone other than the character's voice actor.
[^maximum-dimensions]: Backgrounds which are also used as storyboard images may follow [the storyboard images rule for maximum dimensions](/wiki/Ranking_Criteria#rules.7) instead.
[^normal-vs-addition]: [Hitsounds](/wiki/Beatmapping/Hitsound) consist of an always-present *hitnormal* sample, and any combination of whistle, finish, or clap sample *additions*.

> [^songs-comp-note]: The following guideline does not apply to [Song compilations](/wiki/Beatmap/Song_compilation).

Reviewer: ```suggestion
[^songs-comp-note]: The following guideline does not apply to [song compilations](/wiki/Beatmap/Song_compilation).
```

> [^songs-comp-note]: The following guideline does not apply to [song compilations](/wiki/Beatmap/Song_compilation).

Reviewer: ```suggestion
[^songs-comp-note]: The following rule does not apply to [song compilations](/wiki/Beatmap/Song_compilation).
```
